src/toolkits/feature_engineering/word_trimmer.cpp

```diff
@@ -204,6 +200,12 @@ void word_trimmer_topk_index_mapping(const gl_sarray& src,
           break;
         }
 
+        case flex_type_enum::UNDEFINED:
+          logstream(LOG_WARNING)
```

Comment:
I think we might want to make this a user-facing warning rather than just sending it to a log stream. Not sure what the best way to do that is. For visualization uses, I think we typically just `printf`, but that may not be good from code that could be called programmatically vs. interactively.

Reply:
`cerr` might be enough. Let me check if there's any user facing log stream.

Reply:
@znation now I remove all warnings. Skipping missing value should be an expected behavior. The training phase will decide skip or raise error. For this task, it will raise
```
Please use dropna() to drop rows with missing target values.. operator() from ../src/toolkits/supervised_learning/supervised_learning.cpp at 122'
```
if target or feature columns contain missing values.